recipe/meta.yaml

```diff
@@ -30,12 +30,12 @@ requirements:
   run:
     - python >={{ python_min }}
     - alpha-shapes >=1.1.0
-    - setuptools >=69.0
-    - h5py >=3.10
-    - numpy >=1.26,<2
-    - scipy >=1.11.4
-    - pandas >=2,<2.1
-    - pyzmq >=26.0
+    - setuptools
+    - h5py >=2.10
+    - numpy >=1.20
```

Comment:
Either upstream needs to use `np.exceptions.RankWarning` or we need to pin here to numpy <2.